### PR TITLE
cleanup: Remove duplicated code

### PR DESCRIPTION
**Pull request by vrancurel**
Bucketclient and bucketfile seem to be identical to Arsenal

There was a fix done in Cloudserver's bucketclient more recently but I think it is not necessary to report it to Arsenal. Please review carefully.

### CODE REVIEW

**Review by bennettbuchanan — APPROVED** [review]

**Comment by JianqinWang** [comment]
@bert-e wait

File: lib/metadata/wrapper.js [hunk 1]
```diff
@@ -2,7 +2,6 @@ const MetadataWrapper = require('arsenal').storage.metadata.MetadataWrapper;
 const { config } = require('../Config');
 const logger = require('../utilities/logger');
 const constants = require('../../constants');
-const bucketclient = require('bucketclient');
```
Comment (JianqinWang) [inline lib/metadata/wrapper.js]: From my pov, this is still needed :/ the MetadataWrapper is expecting this as a param

**Review by JianqinWang — CHANGES_REQUESTED** [review]

**Comment by vrancurel** [comment]
I have to modify Arsenal for including bucketclient. Was there a reason not do it @JianqinWang ?

File: lib/metadata/wrapper.js (hunk 1 above)
Comment (vrancurel) [reply lib/metadata/wrapper.js]: question: why ?

**Comment by JianqinWang** [comment]
@vrancurel I think it was because bucketclient has an Arsenal dependency, and @jonathan-gramain or @rahulreddy warned me that having that kind of circular dependency is not good

**Comment by vrancurel** [comment]
There was a name clash between the local 'bucketclient' folder that I removed now and the 'bucketclient' defined in package.json. Please check that everything is correct now.

**Review by JianqinWang — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e reset

**Pull request closed without merging**